test/controllers/session_answers_controller_test.rb

```diff
@@ -51,4 +51,35 @@ def params
       assert_match(/govuk-error-message/, response.body)
     end
   end
+
+  context "GET /:id/clear_session" do
+    setup do
+      get update_session_flow_path(id: flow_name, node_name: nodes[0]), params: params
+    end
+
+    should "be successful when the ext_r option is present and true" do
```

Comment:
Nit: we could be more specific about what successful is here?
```suggestion
    should "redirect to external site when the ext_r option is present and true" do
```

Reply:
And also for the other usages of "successful" on the test descriptions below

Reply:
Hi @theseanything - thanks.  Wording updated.